File: backend/components/metiers/threed/security/docs/samples/access/samples/sso/sample_sso.js
// sample_sso.js
// Exemple d’intégration avancée SSO (Single Sign-On)

const { initiateSsoLogin } = require('../helpers/sso_helper');

const sso = initiateSsoLogin('google', 'https://app.dihya.io/callback');
console.log('SSO Login URL:', sso.url);
console.log('Provider:', sso.provider);
console.log('Initiated At:', sso.initiatedAt);

File: backend/components/metiers/threed/security/docs/samples/access/samples/sso/sample_sso.py
# sample_sso.py
# Exemple d’intégration avancée SSO (Single Sign-On)

from helpers.sso_helper import initiate_sso_login

sso = initiate_sso_login('google', 'https://app.dihya.io/callback')
print('SSO Login URL:', sso['url'])
print('Provider:', sso['provider'])
print('Initiated At:', sso['initiatedAt'])
